backend/Application/Api/GraphQL/Import/EventLogs/CommonParsers.cs

```diff
@@ -50,25 +50,16 @@ public static BigInteger ParseTokenAmount(BinaryReader st)
         /// </summary>
         /// <param name="st"><see cref="BinaryReader"/></param>
         /// <returns>Parsed Address</returns>
-        public static BaseAddress ParseAddress(BinaryReader st)
+        public static Address ParseAddress(BinaryReader st)
```

Comment:
There are test “around” this logic. Those which validates events were correctly parsed. 

It would of course have been nice test only for this part had been present, but I believe the other test “around” this method is ok.